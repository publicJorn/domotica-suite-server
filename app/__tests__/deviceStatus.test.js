const deviceStatus = require('../deviceStatus')

test('Device statuses should be as expected', () =>
  expect(deviceStatus).toMatchSnapshot()
)
