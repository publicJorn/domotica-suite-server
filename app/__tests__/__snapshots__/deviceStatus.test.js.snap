// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Device statuses should be as expected 1`] = `
Object {
  "ERROR": "some generic error",
  "IDENTITY_ERROR": "incorrect identity information",
  "MONITOR_OK": "sensor identity ok",
  "SENSOR_ALERT": "sensor triggered",
  "SENSOR_OK": "default status ok",
  "SENSOR_PENDING": "pending identity",
  "SENSOR_UNCONNECTED": "registered sensor is not connected",
}
`;
